queries.sql

```diff
@@ -9,3 +9,42 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered;
 SELECT * FROM animals WHERE NOT name  = 'Gabumon';
 SELECT * FROM animals WHERE  weight_kg BETWEEN 10.4 AND 17.3;
+
+
+-- project 2
+BEGIN;
+ALTER TABLE animals RENAME COLUMN species TO unspecified;
```

Comment:
You are not necessarily required to rename the species column but to `SET` the column records to unspecified.

e.g
```
...
  UPDATE animals
   SET species = 'unspecified';
...
```